CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Features
 
-* Implemented linear mixed effects with Dirichlet-multinomial distribution ([#2113](https://github.com/scikit-bio/scikit-bio/pull/2113))
+* Implemented linear mixed effects with Dirichlet-multinomial distribution ([#2113](https://github.com/scikit-bio/scikit-bio/pull/2113)).
+* Corrected write function error for embeddings ([#2100](https://github.com/scikit-bio/scikit-bio/pull/2100)).
```

Comment:
Let's move this to the Bug Fixes section

Reply:
@paarth-b Want to do it?